overlays/xtensa_amd_acp_7_0_adsp/gdb/gdb/xtensa-config.c

```diff
@@ -0,0 +1,526 @@
+     XTREG(249,1115, 1, 1, 1,0x001f,0x0006,-2, 6,0x1010,b15,
+               0,0,&xtensa_mask15,0,0,0)
+     XTREG(250,1116, 4, 4, 4,0x2007,0x0006,-2, 6,0x1010,psintlevel,
+               0,0,&xtensa_mask16,0,0,0)
+     XTREG(251,1120, 1, 4, 4,0x2008,0x0006,-2, 6,0x1010,psum,
+               0,0,&xtensa_mask17,0,0,0)
+     XTREG(252,1124, 1, 4, 4,0x2009,0x0006,-2, 6,0x1010,pswoe,
+               0,0,&xtensa_mask18,0,0,0)
+     XTREG(253,1128, 2, 4, 4,0x200a,0x0006,-2, 6,0x1010,psring,
+               0,0,&xtensa_mask19,0,0,0)
+     XTREG(254,1132, 1, 4, 4,0x200b,0x0006,-2, 6,0x1010,psexcm,
+               0,0,&xtensa_mask20,0,0,0)
+     XTREG(255,1136, 2, 4, 4,0x200c,0x0006,-2, 6,0x1010,pscallinc,
+               0,0,&xtensa_mask21,0,0,0)
+     XTREG(256,1140, 4, 4, 4,0x200d,0x0006,-2, 6,0x1010,psowb,
+               0,0,&xtensa_mask22,0,0,0)
+     XTREG(257,1144, 6, 4, 4,0x200e,0x0006,-2, 6,0x1010,mpunumentries,
+               0,0,&xtensa_mask23,0,0,0)
+     XTREG(258,1148, 4, 4, 4,0x2013,0x0006,-2, 6,0x1010,dbnum,
+               0,0,&xtensa_mask24,0,0,0)
+     XTREG(259,1152, 1, 4, 4,0x2016,0x0006, 1, 6,0x1010,ae_overflow,
+               0,0,&xtensa_mask25,0,0,0)
+     XTREG(260,1156,14, 4, 4,0x2017,0x0006, 1, 6,0x1010,ae_sar,
+               0,0,&xtensa_mask26,0,0,0)
+     XTREG(261,1160, 1, 4, 4,0x2018,0x0006, 1, 6,0x1010,ae_cwrap,
+               0,0,&xtensa_mask27,0,0,0)
+     XTREG(262,1164, 4, 4, 4,0x2019,0x0006, 1, 6,0x1010,ae_bitptr,
+               0,0,&xtensa_mask28,0,0,0)
+     XTREG(263,1168, 4, 4, 4,0x201a,0x0006, 1, 6,0x1010,ae_bitsused,
+               0,0,&xtensa_mask29,0,0,0)
+     XTREG(264,1172, 4, 4, 4,0x201b,0x0006, 1, 6,0x1010,ae_tablesize,
+               0,0,&xtensa_mask30,0,0,0)
+     XTREG(265,1176, 4, 4, 4,0x201c,0x0006, 1, 6,0x1010,ae_first_ts,
+               0,0,&xtensa_mask31,0,0,0)
+     XTREG(266,1180,27, 4, 4,0x201d,0x0006, 1, 6,0x1010,ae_nextoffset,
+               0,0,&xtensa_mask32,0,0,0)
+     XTREG(267,1184, 1, 4, 4,0x201e,0x0006, 1, 6,0x1010,ae_searchdone,
+               0,0,&xtensa_mask33,0,0,0)
+     XTREG(268,1188, 8, 4, 4,0x201f,0x0006, 1, 6,0x1010,ae_zbiasv8,
+               0,0,&xtensa_mask34,0,0,0)
+     XTREG(269,1192, 8, 4, 4,0x2020,0x0006, 1, 6,0x1010,ae_zbiasc8,
+               0,0,&xtensa_mask35,0,0,0)
+     XTREG(270,1196, 2, 4, 4,0x2021,0x0006, 1, 6,0x1010,roundmode,
+               0,0,&xtensa_mask36,0,0,0)
+     XTREG(271,1200, 1, 4, 4,0x2022,0x0006, 1, 6,0x1010,invalidflag,
+               0,0,&xtensa_mask37,0,0,0)
+     XTREG(272,1204, 1, 4, 4,0x2023,0x0006, 1, 6,0x1010,divzeroflag,
+               0,0,&xtensa_mask38,0,0,0)
+     XTREG(273,1208, 1, 4, 4,0x2024,0x0006, 1, 6,0x1010,overflowflag,
+               0,0,&xtensa_mask39,0,0,0)
+     XTREG(274,1212, 1, 4, 4,0x2025,0x0006, 1, 6,0x1010,underflowflag,
+               0,0,&xtensa_mask40,0,0,0)
+     XTREG_END
+   };
+   
+   
+   
+   #ifdef XTENSA_CONFIG_INSTANTIATE
+   XTENSA_CONFIG_INSTANTIATE(rmap,16)
```

Comment:
please see b7a0a3da90832a5b114c1da31bf462c1ff1bc3eb for how this should look like, use xtensa_rmap here.

Ana